server/chat-plugins/lottery.ts

```diff
@@ -89,14 +93,16 @@ export const commands: ChatCommands = {
 			}
 			return this.parse(`/join view-lottery-${room.id}`);
 		},
-		create(target, room, user) {
+		edit: 'create',
+		create(target, room, user, connection, cmd) {
 			if (!this.can('declare', null, room)) return;
 			if (room.battle || !room.chatRoomData) {
 				return this.errorReply('This room does not support the creation of lotteries.');
 			}
 			const lottery = lotteries[room.id];
-			if (lottery && lottery.running) {
-				return this.errorReply(`There is already a lottery running in this room.`);
+			const edited = lottery && lottery.running;
+			if (cmd === 'edit' && !target && lottery) {
+				return this.sendReplyBox(`<strong>Max winners: </strong>${lottery.maxWinners}<br /><strong>Name: </strong>${lottery.name}<br /><strong>Markup: </strong>${Chat.html`${lottery.markup}`}`);
```

Comment:
This is in the wrong format - ideally you should be able to copy and paste the exact contents that are returned from a `code` block to be able to reproduce the existing lottery like with other commands that allow for editing. cf https://github.com/Zarel/Pokemon-Showdown/blob/master/server/chat-commands.js#L1287  and https://github.com/Zarel/Pokemon-Showdown/blob/master/server/chat-commands.js#L1341